### DISCUSSION TITLE
RPi4 + Raspbee II -  Z2MQTT 502: Bad Gateway

### DISCUSSION THREAD

**Discussion by snoop69x** [Koenkk/zigbee2mqtt#13724] [Devices]
Hi, I'm asking the experts here for help. I don't know what else to set so that Z2MQTT doesn't give me the error 502: Bad Gateway. Just set /dev/ttyAMA0 in the deCONZ plugin and it works, but Z2MQTT still nothing.

[Z2MQTT options](http://imgway.cz/v/4Ksl.jpg)
[Z2MQTT log](http://imgway.cz/v/4Ksj.jpg)

HW:
Raspberry 4 8GB
Raspbee II

SW:
Home Assistant 2022.8.7
Zigbee2MQTT 1.27.0-1
Mosquitto broker 6.1.2

**Comment by lux73** [comment]
the Log says `Resource busy` - so it seems the Stick is anyway using from other Plugin/Software

try to disable/remove this Software

**Comment by snoop69x** [comment]
log after uninstalling the deCONZ add-on
[Z2MQTT log](http://imgway.cz/v/4Kss.jpg)
